compiler/rustc_mir/src/interpret/validity.rs

```diff
@@ -214,6 +214,11 @@ struct ValidityVisitor<'rt, 'mir, 'tcx, M: Machine<'mir, 'tcx>> {
 }
 
 impl<'rt, 'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> ValidityVisitor<'rt, 'mir, 'tcx, M> {
+    fn allow_uninit_and_ptr(&self) -> bool {
```

Comment:
I should probably rename this to something like `allow_uninit_and_ptr_numbers()`.